dev/documentation/en/resources.md

```diff
@@ -23,4 +23,16 @@ A CLI script to [generate submissions for import](https://gist.github.com/NateWr
 
 ## Generate metrics
 
-A CLI tool to [generate fake metrics data](https://gist.github.com/NateWr/32322ca420a0abb37c035506dd51601d). Creates fake stats entries for each submission in your instance.
\ No newline at end of file
+A CLI tool to [generate fake metrics data](https://gist.github.com/NateWr/32322ca420a0abb37c035506dd51601d). Creates fake stats entries for each submission in your instance.
```

Comment:
This tool should probably be combined with https://github.com/pkp/pkp-lib/blob/master/tools/generateTestMetrics.php (which is used for https://github.com/pkp/ojs/blob/master/cypress/tests/integration/Statistics.spec.js).

Reply:
Oh yeah. Hah, these are almost identical. I'll try to merge the feature-sets.

Reply:
Will merge once the tests pass.

PRs:
https://github.com/pkp/pkp-lib/pull/6030
https://github.com/pkp/ojs/pull/2818